packages/adapter-cloudflare-workers/files/entry.js

```diff
@@ -57,7 +57,10 @@ export default {
 		const filename = stripped_pathname.substring(1);
 		if (filename) {
 			is_static_asset =
-				manifest.assets.has(filename) || manifest.assets.has(filename + '/index.html');
+				manifest.assets.has(filename) ||
+				manifest.assets.has(filename + '/index.html') ||
+				Object.prototype.hasOwnProperty.call(manifest._.server_assets, filename) ||
+				Object.prototype.hasOwnProperty.call(manifest._.server_assets, filename + '/index.html');
```

Comment:
```suggestion
				filename in manifest._.server_assets ||
				filename + '/index.html' in manifest._.server_assets);
```

Reply:
Because I've written it almost half a year ago, I can't recall why I used `hasOwnProperty` 😅
I have vague recollection that I tried `in` and something didn't work, but I tested it again today, and it seamed to work as intended, so I've changed it (in all 3 adapters: Cloudflare, Cloudflare-workers and netlify).